infrastructure-playbooks/add-mon.yml

```diff
@@ -0,0 +1,67 @@
+---
+# This playbook is used to add a new MON to
+# an existing cluster. It can run from any machine. Even if the fetch
+# directory is not present it will be created.
+#
+# Ensure that all monitors are present in the mons
+# group in your inventory so that the ceph configuration file
+# is created correctly for the new OSD(s).
+- hosts: mons
+  gather_facts: false
+  vars:
+    delegate_facts_host: true
+  pre_tasks:
+    - name: gather facts
+      setup:
+      when: not delegate_facts_host | bool
+    - name: gather and delegate facts
+      setup:
+      delegate_to: "{{ item }}"
+      delegate_facts: true
+      with_items: "{{ groups['mons'] }}"
+      run_once: true
+      when: delegate_facts_host | bool
+  tasks:
+    - import_role:
```

Comment:
you've pushed recently a patch in add-osds.yml to avoid hardcoding the group name (L21)
you might want to do the same thing here.